packages/fxa-settings/src/components/Tooltip/index.tsx

```diff
@@ -31,9 +33,13 @@ export const Tooltip = ({
     bottom: 'top',
   }[position];
 
+  function formatDataTestId(id: string) {
```

Comment:
Not a huge deal, but this is duplicated, we could move it into [lib/utilities.ts](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/utilities.ts)